apps/frontend/src/app/PageCharacter/CharacterDisplay/Tabs/TabTalent.tsx

```diff
@@ -54,9 +54,10 @@ export default function CharacterTalentPane() {
 
   const theme = useTheme()
   const grlg = useMediaQuery(theme.breakpoints.up('lg'))
+  const maxConstellationCount = 6
```

Comment:
nit: move this to https://github.com/frzyc/genshin-optimizer/blob/master/libs/consts/src/character.ts